editor/src/messages/portfolio/menu_bar/menu_bar_message_handler.rs

```diff
@@ -254,7 +254,7 @@ impl LayoutHolder for MenuBarMessageHandler {
 				no_active_document,
 				MenuBarEntryChildren(vec![vec![MenuBarEntry {
 					label: "Clear Artboards".into(),
-					action: MenuBarEntry::create_action(|_| DialogMessage::RequestComingSoonDialog { issue: None }.into()),
+					action: MenuBarEntry::create_action(|_| GraphOperationMessage::ClearArtboards.into()),
```

Comment:
This was disabled because it crashes (as it wasn't ported to document nodes). And with this PR (at least on my computer) this still does crash. So I'll revert this line for now so it doesn't hold up the rest of the PR which I'm merging now.